wiki/en/Privacy-Statement.md

```diff
@@ -7,41 +7,43 @@ permalink: "/wiki/Privacy-Statement"
 
 # Privacy Statement
 
-Please note that the English version of this privacy statement is the original and, as such, the binding version. To access the English version, go to the top/top right of this page and click on the "en" link.
-The following only applies to the version you can download from this website, [jamulus.io](https://jamulus.io). 
+**The English version of this document represents the official privacy statement for the _current version_ of the software available on this website. If there is a conflict between the English language version and any translation, the English version shall apply. Earlier versions may have had different privacy terms which are no longer applicable or supported. Users are encouraged to use the most current version of the Jamulus software.**
+ 
 
 ## Definition of Terms
 
-- "**Server**" The Jamulus Server software, as opposed to the host machine/OS it is running on.
-- "**Client**" The Jamulus software used to connect to a Server
+- "**Server**" The Jamulus Server software running on a remote computer to which you will connect.
```

Comment:
This assumes I'm not a server operator.  "Server" should mean the same whether or not I'm running one.